### ISSUE TITLE
Still need waitFor?

### ISSUE THREAD

**Issue by KyleAMathews** [reflux/refluxjs#18]
I'm really liking in general the simplifications you've made over Facebook's Flux implementation. But one possible weakness with your approach of _listening_ to stores vs. _waiting_ for stores is stores lower in the hierarchy would lose access to the raw action data. This would be a problem if the top-level store only preserves a filtered view of the raw action data. A lower-level store could get around this to some degree by also listening to an action and storing its data and then later combining that with the higher-level store's data but then there could be problem ensuring matching up data.

So in short, I see value still in having a way of doing ordered event callbacks.
